core/js/src/main/scala/org/log4s/Logger.scala

```diff
@@ -1,10 +1,23 @@
 package org.log4s
 
 case class Logger(name: String) {
-  def trace(s: => String): Unit = {}
-  def debug(s: => String): Unit = {}
-  def info(s: => String): Unit = {}
-  def warn(s: => String): Unit = {}
-  def error(t: Throwable)(s: => String): Unit = {}
-  def error(t: Any)(s: => String): Unit = {}
+  def trace(s: => String): Unit =
+    println(s)
+
+  def debug(s: => String): Unit =
+    println(s)
```

Comment:
These could be extremely verbose.  We might want to have a *gasp* global threshold.